slybot/slybot/plugins/scrapely_annotations/extraction/pageparsing.py

```diff
@@ -25,6 +33,129 @@ def to_template(self, descriptors=None):
                                   self.html_page.page_id, self.ignored_regions,
                                   self.extra_required_attrs, descriptors)
 
+    @staticmethod
```

Comment:
Looking at the scrapely code I prefer to keep it like this. It's quite easy to see what has been overwritten and this could even be backported to scrapely as is.
It's also unlikely that there will be any change to the TemplateParser in scrapely so this won't be a problem anyway.